src/DEV-2/DEV-2/Checker.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DEV_2
+{
+    // Check input arithmetic expression
+    class Checker
+    {
+        private bool correctExpression = true;
+
+        // If  input data correct return true 
+        public bool checkInputData(string data)
+        {
+            // Remove " " symbols
+            data.Replace(" ", "");
+
+            // Check not valid symbols
+            // incorrect positions of operations
+            // first and last symbols
+            if ((checkSymbols(data) == false) || (checkOperationsPosition(data) == false) || (checkFirstLastSymbol(data) == false))
+            {
+                Console.Write("Error input");
+                correctExpression = false;
+            }
+
+            // Check Brackets
+            if (checkBrackets(data) == false)
+            {
+                Console.Write("Numbers of ( and ) is not the same");
+                correctExpression = checkBrackets(data);
+            }
+
+            return correctExpression;
+        }
+
+        // Check number of "(" and ")"
+        private bool checkBrackets(string data)
+        {
+            bool check = true;
+            int numberofLeft = 0;
+            int numberofRight = 0;
+
+            foreach (char c in data)
+            {
+                if (c == '(')
+                    numberofLeft++;
+                if (c == ')')
+                    numberofRight++;
+            }
+            if (numberofLeft != numberofRight)
+                check = false;
+
+            return check;
+        }
+
+        // Check first and last symbols in data 
+        private bool checkFirstLastSymbol(string data)
+        {
+            bool check = true;
```

Comment:
Do not use separate comparison, create string of valid symbols and check any input symbol for it